docs/source/storage_resizing.md

```diff
@@ -0,0 +1,76 @@
+# Resizing storage in Scylla Cluster
+
+Unfortunately changes in storage are currently not supported - it has to be done manually. 
+
+1. List Scylla Cluster 
+    ```
+    k get pods -n scylla                                                      
+                 
+    NAME                                    READY   STATUS    RESTARTS   AGE
+    simple-cluster-us-east-1-us-east-1a-0   2/2     Running   0          12m
+    simple-cluster-us-east-1-us-east-1a-1   2/2     Running   0          11m
+    simple-cluster-us-east-1-us-east-1a-2   2/2     Running   0          9m27s
+    ``` 
+2. Orphan delete Scylla Cluster
+    ```
+    kubectl delete -n scylla scyllacluster simple-cluster --cascade='orphan'
+   
+    scyllacluster.scylla.scylladb.com "simple-cluster" deleted
+    ```
+3. List Statefulsets 
+    ```
+    k get statefulset -n scylla
+   
+    NAME                                  READY   AGE
+    simple-cluster-us-east-1-us-east-1a   3/3     12m
+    ```
+4. Orphan delete Statefulsets
+    ```
+    k delete -n scylla statefulset/simple-cluster-us-east-1-us-east-1a --cascade='orphan'
+   
+    statefulset.apps "simple-cluster-us-east-1-us-east-1a" deleted
+    ```
+5. List PVCs
+    ```
+    kubectl get pvc -n scylla 
+    
+    NAME                                         STATUS   VOLUME                                     CAPACITY   ACCESS MODES   STORAGECLASS   AGE
+    data-simple-cluster-us-east-1-us-east-1a-0   Bound    pvc-c0182451-1545-47d7-8784-d346d512cd4e   1Gi        RWO            standard       15m
+    data-simple-cluster-us-east-1-us-east-1a-1   Bound    pvc-41d62296-8848-48ea-a1e9-c01e1f72607a   1Gi        RWO            standard       12m
+    data-simple-cluster-us-east-1-us-east-1a-2   Bound    pvc-bbe46dba-c209-4f78-9ff8-e2970508235c   1Gi        RWO            standard       12m
+    ```
+6. Change storage request 
+    ```
+    k -n scylla patch pvc data-simple-cluster-us-east-1-us-east-1a-0 -p '{"spec":{"resources":{"requests":{"storage":"2Gi"}}}}'
+    
+    persistentvolumeclaim/data-simple-cluster-us-east-1-us-east-1a-0 patched
+    ```
+    ```
+    k -n scylla patch pvc data-simple-cluster-us-east-1-us-east-1a-1 -p '{"spec":{"resources":{"requests":{"storage":"2Gi"}}}}'
+    
+    persistentvolumeclaim/data-simple-cluster-us-east-1-us-east-1a-1 patched
+    ```
+    ```
+    k -n scylla patch pvc data-simple-cluster-us-east-1-us-east-1a-2 -p '{"spec":{"resources":{"requests":{"storage":"2Gi"}}}}'
+    
+    persistentvolumeclaim/data-simple-cluster-us-east-1-us-east-1a-2 patched
+    ```
+
+    If the following error occurred: 
```

Comment:
```suggestion
    If the PVC change is denied because the PVC doesn't support volume expansion you need replace every node following the procedure described in ...
```